pkg/server/gen/server.pb.go

```diff
@@ -4,7 +4,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.28.1
-// 	protoc        v3.17.3
+// 	protoc        v3.21.9
```

Comment:
Is this intentional? I checked this branch out locally and ran `make docker/gen/server` and got a diff showing a switch back to `v3.17.3`

Reply:
It was not intentional, I ran `make gen/server` instead of `make docker/gen/server`. But, fixed!